README.md

```diff
@@ -141,21 +141,41 @@ define(['doc'], function(doc) {
 });
 ```
 
+
+#### prepend
+`.prepend(element)`
+
+###### Description:
+Prepend the DOM element as child of the Doc object
+
+###### Parameters:
+> element: Element //The element you want to prepend as a child of your selector.
+
+###### Sample:
+``` js
+define(['doc'], function(doc) {
+	var div = document.createElement('div');
+	doc('body').prepend(div); //Prepend the new DOM element as child of the Doc object
+	doc('body').prepend($('div.content').first()); //Prepend the exist DOM element as child of the Doc object
+	doc('body').prepend('<p>Text here</p>'); //Prepend text HTML as child of the Doc object
```

Comment:
It is lacking the example `doc('body').prepend($('.some-element'));`, where you prepend a doc element.